blockchain/cost.go

```diff
@@ -78,6 +84,14 @@ func GetTransactionCost(tx *btcutil.Tx) int64 {
 	return int64((baseSize * (WitnessScaleFactor - 1)) + totalSize)
 }
 
+// GetMsgTxCost is the same as GetTransactionCost but doesn't use btctil.Tx
+func GetMsgTxCost(tx *wire.MsgTx) int64 {
```

Comment:
`GetTransactionCost` can just call this function directly with the the `tx's internal `wire.MsgTx`. This would eliminate the duplication in this function.